### ISSUE TITLE
Refactor HDR RECEIVED_FROM and DATA_TRANSFER attributes

### ISSUE THREAD

**Issue by pedrogit** [edwardsmarc/CASFRI#416]
Some HDR attributes contain more than one type of information. e.g. RECEIVED_FROM contains a mixture of URLs and contact names. I would like to put the names in the CONTACT_INFO attribute and rename RECEIVED_FROM to ACQUISITION_LINK.

I would also rename DATA_TRANSFER to ACQUISITION_TYPE to match ACQUISITION_DATE. I also want to replaced its values with standardized codes.

Any objections?

I do that while doing a major clean up of the inventory_list_cas05.csv. I properly name some attributes (not part of HDR) and fill missing values with something more meaningful (NOT_APPLICABLE and UNKNOWN_VALUES).

**Comment by pedrogit** [comment]
I would also rename INVENTORY_MANUAL to INVENTORY_MANUAL_NAME.

**Comment by pedrogit** [comment]
Some attributes values are to be replaced with codes and those codes need to be added to the spec with their meaning:

SRC_DATA_FORMAT, ACQUISITION_TYPE, REDISTRIBUTION, LICENSE_AGREEMENT and PHOTO_YEAR_TYPE.

**Comment by pedrogit** [comment]
I would propably also add PRODUCTION_YEAR to reflect the year that we sometimes find in the name of the dataset (e.g. BC see discussion in #399) so we could make sence of it. We would then have a more complete set of dates: 

- PHOTO_YEAR_START and PHOTO_YEAR_END: Interval years of the photos used for photointerpretation.
- PRODUCTION_YEAR_END: Year the dataset production was finished. (some datasets takes years to produce)
- PUBLICATION_DATE: Date the dataset was published on the web site where the CASFRI project acquired it.
- ACQUISITION_DATE: Date the dataset was acquired by the CASFRI project.

We might want to add  PRODUCTION_YEAR_BEGIN... Or make PRODUCTION_YEARS (with an S) an interval when necessary.

**Comment by pedrogit** [comment]
It would also be necessary to review the values in the STANDARD_VERSION and STANDARD_REVISION attributes since mst are left empty and some are just duplicates of the other.

**Comment by MelinaHoule** [comment]
About PRODUCTION_YEAR_XX, not sure we will have that information for most of the inventory.  I don't remember seing a range for production year. It is  usually represented by single year. Before adding a column, maybe we should investigate if the column would be filled with something or always empty. Maybe with new inventory, metadata are more complete and it is relevant. Maybe...

**Comment by pedrogit** [comment]
Right. I would add it mostly for new inventories and because it is a usefull info to have. Not with the goal of filling it for old inventories. When the info is available but the field is not present  we tend to lose it and it get lost forever. And I would prefer PRODUCTION_YEARS a an interval so we can store a single year or an interval.

**Comment by pedrogit** [comment]
I will actually rename INVENTORY_MANUAL to STANDARD_MANUAL_NAME

**Comment by pedrogit** [comment]
All changes above were made.

Now it would be a good idea to review all the attributes related to licensing: DATA_AVAILABILITY, REDISTRIBUTION, PERMISSION, and LICENSE_AGREEMENT. Some are not clearly defined and some store quite similar information.
